sidebars.js

```diff
@@ -14,30 +14,45 @@ module.exports = {
       items: [
         {
           type: "category",
-          label: "Install Okteto",
-          link: { type: 'doc', id: 'get-started/install/index' },
+          label: "For Platform Engineers",
           items: [
-            "get-started/install/amazon-eks",
-            "get-started/install/civo",
-            "get-started/install/digitalocean-doks",
-            "get-started/install/google-gke",
-            "get-started/install/microsoft-aks"
+            {
+              type: "category",
+              label: "Install Okteto",
+              link: { type: 'doc', id: 'get-started/install/index' },
+              items: [
+                "get-started/install/amazon-eks",
+                "get-started/install/civo",
+                "get-started/install/digitalocean-doks",
+                "get-started/install/google-gke",
+                "get-started/install/microsoft-aks"
+              ]
+            },
+            "get-started/install-okteto-cli",
+            {
+              type: "category",
+              label: "Deploy your app",
+              link: { type: 'doc', id: 'get-started/deploy-your-app/index' },
+              items: [
+                "get-started/deploy-your-app/deploy",
+                "get-started/deploy-your-app/dependencies",
+                "get-started/deploy-your-app/endpoints",
+                "get-started/deploy-your-app/build",
+              ]
+            },
+            "get-started/next-steps",
           ]
         },
-        "get-started/install-okteto-cli",
         {
           type: "category",
-          label: "Deploy your app",
-          link: { type: 'doc', id: 'get-started/deploy-your-app/index' },
+          label: "For Developers",
```

Comment:
Would it make sense to have a separate "Install CLI" section in the developer's guide, or should we keep it as the fourth step of the quickstart? Having a dedicated section could be useful for linking directly from the UI or other resources. Perhaps we could create a general section that's common for both personas and also include it as a step for platform engineers.

Reply:
I agree with breaking this out into its own page, especially for linking, but not sure where to put it. 
Another option: @rlamana is it possible to create a variable that is the content for this CLI page so that we can have a shared source of truth for the Developer and Platform CLI installation instructions?

Reply:
I'm going to merge this @rlamana and I will create a new PR exploring creating a dedicated page for the CLI installation

Reply:
Sorry I didn't see this earlier. We can have some sort of shared variable, if I recall we've done it somewhere else. We can do it in a different PR.